test/dynamo/test_autograd_function.py

```diff
@@ -526,6 +528,7 @@ def forward(self, x):
     # I pulled all of these test cases from test_autograd.py
     # In the future, we should make the Dynamo test suite actually
     # run on test_autograd.py (it's disabled right now) and delete these.
+    @unittest.skipIf(sys.version_info >= (3, 12), "invalid free in 3.12+")
```

Comment:
Do you think it would be good to create an issue that will track all failures that still exist on 3.12?